addons/account/models/ir_actions_report.py

```diff
@@ -1,12 +1,16 @@
 # -*- coding: utf-8 -*-
+import base64
 import io
 import textwrap
 from collections import OrderedDict
+from PIL import Image, ImageFile
 
 from odoo import models, _
 from odoo.exceptions import UserError
 from odoo.tools import pdf
 
+# Allow truncated images
+ImageFile.LOAD_TRUNCATED_IMAGES = True
```

Comment:
is it really necessary? this is patching the module for all of Odoo code. Usually if it's only necessary in one method we try to change the value only for that method (by using a parameter, or modifying something in the object that Image.open returns for example)